app/views/payment/PaymentStudentLicensePurchaseView.vue

```diff
@@ -37,7 +37,9 @@
 
 <script>
 import { loadStripe } from '@stripe/stripe-js'
-const stripePromise = loadStripe('pk_test_BqKtc6bIKPn6FeSA4GhuRrwT');
+const isProd = document.location.href.includes('codecombat.com')
```

Comment:
does local key only use in localhost? can use prod key in staging.codecombat.com ? is so , i think this is fine